src/printing-job/printing-job.resolver.ts

```diff
@@ -0,0 +1,17 @@
+import { Inject } from '@nestjs/common';
+import { Query, Resolver } from '@nestjs/graphql';
+import { PrintingJobDto } from './dto/printing-job.dto';
+import { PrintingJobService } from './printing-job.service';
+
+@Resolver('PrintingJob')
+export class PrintingJobResolver {
+  constructor(
+    @Inject(PrintingJobService)
```

Comment:
There is no need to inject the service (in any of the resolvers). Dependency Injection will do it for you.